x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/ml_model_health.tsx

```diff
@@ -15,28 +15,40 @@ import { FormattedMessage } from '@kbn/i18n-react';
 import { MlModelDeploymentState } from '../../../../../../common/types/ml';
 import { TrainedModelState } from '../../../../../../common/types/pipelines';
 
+const modelNotDownloadedText = i18n.translate(
+  'xpack.enterpriseSearch.inferencePipelineCard.modelState.notDownloaded',
+  {
+    defaultMessage: 'Not deployed',
```

Comment:
The wording of the keys and labels differ. This is a known discrepancy and will be addressed later in 8.13+ while unifying the ML UX.